frontend/src/app/trees/forests/tree-guidelines/sidebar-view.component.html

```diff
@@ -3,49 +3,8 @@
-          <ul class="usa-sidenav-sub_list" *ngIf="util.currentSection && util.currentSubSection && util.currentSection.startsWith('cutting-instructions') && util.currentSubSection.startsWith('cutting-instructions')">
-            <li>
-              <a href="#" id="cutting-instructions-before-you-cut-link" [ngClass]="{'usa-current': util.currentSection && util.currentSection  === 'cutting-instructions-before-you-cut'}" (click)="util.gotoHashtag('cutting-instructions-before-you-cut', $event, true)">Before you cut</a>
-            </li>
-            <li>
-              <a href="#" id="cutting-instructions-when-you-cut-link" [ngClass]="{'usa-current': util.currentSection && util.currentSection  === 'cutting-instructions-when-you-cut'}" (click)="util.gotoHashtag('cutting-instructions-when-you-cut', $event, true)">When you cut</a>
-            </li>
-          </ul>
-        </li>
-        <li>
-          <a href="#" id="need-to-know-link" [ngClass]="{'usa-current': util.currentSection && util.currentSection.startsWith('need-to-know')}" (click)="util.gotoHashtag('need-to-know', $event)">Rules to know</a>
-        </li>
-        <li>
-          <a href="#" id="trip-planning-link" [ngClass]="{'usa-current': util.currentSection && util.currentSection  === 'trip-planning'}" (click)="util.gotoHashtag('trip-planning', $event)">How to plan your trip</a>
-        </li>
-        <li>
-          <a href="#" id="tree-locations-link" [ngClass]="{'usa-current': util.currentSection && util.currentSection.startsWith('tree-locations')}" (click)="util.gotoHashtag('tree-locations', $event, true)">Where to find your tree</a>
-          <ul class="usa-sidenav-sub_list" *ngIf="util.currentSection && util.currentSubSection && util.currentSection.startsWith('tree-locations') && util.currentSubSection.startsWith('tree-locations')">
-            <li>
-              <a href="#" id="tree-locations-prohibited-link" [ngClass]="{'usa-current': util.currentSection && util.currentSection  === 'tree-locations-prohibited'}" (click)="util.gotoHashtag('tree-locations-prohibited', $event, true)">Prohibited areas and restrictions</a>
-            </li>
-            <li>
-              <a href="#" id="tree-locations-allowed-link" [ngClass]="{'usa-current': util.currentSection && util.currentSection  === 'tree-locations-allowed'}" (click)="util.gotoHashtag('tree-locations-allowed', $event, true)">Cutting area maps</a>
-            </li>
-          </ul>
-        </li>
-        <li>
-          <a href="#" id="contact-information-link" [ngClass]="{'usa-current': util.currentSection && util.currentSection  === 'contact-information'}" (click)="util.gotoHashtag('contact-information', $event)">Contact us</a>
-        </li>
+    <app-sidebar mobileMenu="true" [items]="sidebarItems"  class="usa-width-one-third usa-layout-docs-sidenav"></app-sidebar>
```

Comment:
Can this now just be in the main component for this page since its generated by the central sidebar component.

Reply:
We have added a task in #142 to move the sidebar component into tree-guidelines.component @lauraGgit